include/kodlab_mjbots_sdk/mjbots_hardware_interface.h

```diff
@@ -35,18 +35,18 @@ struct RealtimeParams {
  * @brief A class that allows interaction with the Mjbots Moteus Motor Controllers
  * 
  */
-class MjbotsRobotInterface  {
+class MjbotsHardwareInterface  {
  public:
 
   /*!
    * @brief constructs an mjbots_robot_interface to communicate with a collection of moeteusses
```

Comment:
Update comment

Reply:
Resolved in 8962188d140e046fbc1ade97323bd3b6774e975c.